### PR TITLE
Barebones gaming app

### LINKED ISSUE #1205: Gaming UI Setup
The gaming SDK needs a UI so that the users of this SDK can pay.
- Copy/paste/adapt from the storage-ui repo
- landing page is the `Settings` page
- The API Keys section should remain
- render building on PR

### PR DESCRIPTION
**Pull request by FSM1**
Closes #1205

### CODE REVIEW

File: packages/gaming-ui/src/Components/GamingRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  SettingsRoot: "/settings",
+  Settings: (path: SettingsPath) => `/settings/${path}`,
+  PrivacyPolicy: "https://files.chainsafe.io/privacy-policy",
+  Terms: "https://files.chainsafe.io/terms-of-service",
+  ChainSafe: "https://chainsafe.io/"
+}
+
+
+
+const StorageRoutes = () => {
```
Comment (RyRy79261) [inline packages/gaming-ui/src/Components/GamingRoutes.tsx]:
```suggestion
const GamingRoutes = () => {
```

File: packages/gaming-ui/src/Components/GamingRoutes.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        isAuthorized={!isLoggedIn}
+        component={LoginPage}
+        redirectPath={ROUTE_LINKS.SettingsRoot}
+        redirectToSource
+      />
+    </Switch>
+  )
+}
+
+export default StorageRoutes
```
Comment (RyRy79261) [inline packages/gaming-ui/src/Components/GamingRoutes.tsx]:
```suggestion
export default GamingRoutes
```

File: packages/gaming-ui/src/Components/Modules/ApiKeys.tsx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+      "& svg": {
+        fill: constants.fileSystemItemRow.menuIcon
+      }
+    }
+  })
+)
+
+const ApiKeys = () => {
+  const classes = useStyles()
+  const { gamingApiClient: storageApiClient } = useGamingApi()
```
Comment (RyRy79261) [inline packages/gaming-ui/src/Components/Modules/ApiKeys.tsx]: Rational in keeping the storage prefix?

**Review by RyRy79261 — COMMENTED** [review]
Just some minor things, still need to run the code

File: packages/gaming-ui/src/Components/Modules/LoginModule.tsx [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
+          </div>
+        </section>
+        <Footer/>
+      </div>
+    )}
+
+  const WalletConnection = () => {
+    return (
+      <section className={classes.connectingWallet}>
+        <Typography variant='h2'><Trans>Connect Wallet to Storage</Trans></Typography>
```
Comment (RyRy79261) [inline packages/gaming-ui/src/Components/Modules/LoginModule.tsx]:
```suggestion
        <Typography variant='h2'><Trans>Connect Wallet to Gaming</Trans></Typography>
```

**Review by RyRy79261 — APPROVED** [review]
Looks like its working, just some Storage name artifacts lying around

File: packages/gaming-ui/src/Components/Modules/ApiKeys.tsx (hunk 3 above)
Comment (FSM1) [reply packages/gaming-ui/src/Components/Modules/ApiKeys.tsx]: Nope, fixed now. Was a rename all that went wrong.

**Review by tanmoyAtb — APPROVED** [review]
Awesome!
🎆

**Pull request merged**